vm/src/stdlib/imp.rs

```diff
@@ -5,46 +5,48 @@ use crate::obj::objmodule::PyModuleRef;
 use crate::obj::objstr;
 use crate::obj::objstr::PyStringRef;
 use crate::pyobject::{BorrowValue, ItemProtocol, PyObjectRef, PyResult};
-#[cfg(not(target_arch = "wasm32"))]
-use crate::stdlib::thread::RawRMutex;
 use crate::vm::VirtualMachine;
 
-#[cfg(not(target_arch = "wasm32"))]
-static IMP_LOCK: RawRMutex = RawRMutex::INIT;
+#[cfg(feature = "threading")]
+mod lock {
+    use crate::pyobject::PyResult;
+    use crate::stdlib::thread::RawRMutex;
+    use crate::vm::VirtualMachine;
 
-fn imp_extension_suffixes(vm: &VirtualMachine) -> PyResult {
-    Ok(vm.ctx.new_list(vec![]))
-}
+    pub(super) static IMP_LOCK: RawRMutex = RawRMutex::INIT;
```

Comment:
Hmm, this should probably be in the global state for the vm.

Reply:
agree